### ISSUE TITLE
Warning about security issue in submodule

### ISSUE THREAD

**Issue by nathanpeck** [google/google-api-nodejs-client#460]
This module has a dependency with a security warning:

```
npm WARN deprecated jws@0.0.2: Security update: Versions below 3.0.0 are deprecated.
```

The [changelog](https://github.com/brianloveswords/node-jws/blob/master/CHANGELOG.md) has more details about that security warning:

```
This mitigates a critical security flaw in the library which would allow an attacker
to generate signatures with arbitrary contents that would be accepted by jwt.verify.
```

Here is the dependency tree showing how this vulnerable submodule is being included in the google api client:

```
"googleapis": {
      "version": "2.1.1",
      "from": "googleapis@2.1.1",
      "resolved": "https://registry.npmjs.org/googleapis/-/googleapis-2.1.1.tgz",
      "dependencies": {
        "gapitoken": {
          "version": "0.1.4",
          "from": "gapitoken@>=0.1.2 <0.2.0",
          "resolved": "https://registry.npmjs.org/gapitoken/-/gapitoken-0.1.4.tgz",
          "dependencies": {
            "jws": {
              "version": "0.0.2",
              "from": "jws@0.0.2",
              "resolved": "https://registry.npmjs.org/jws/-/jws-0.0.2.tgz",
```

I'd appreciate it if this client was updated to correct this, as it isn't very nice to see security warnings when doing `npm install`, especially originating from the google api client.

**Comment by JustinBeckwith** [comment]
This was addressed in #522

**Comment by jmdobry** [comment]
Though we have yet to make a new release, so stay tuned.
